x-pack/plugins/cases/common/index.ts

```diff
@@ -7,3 +7,4 @@
 
 export * from './constants';
 export * from './api';
+export * from './ui/types';
```

Comment:
Why the ui types are being exported? Will they be used by another plugin?

Reply:
yes, security solution uses the types on the UI

Reply:
I see! Thanks!